### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by SajanAlexander-okta**
Updates the provider docs to emphasize OPA over ASA

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -15,14 +17,14 @@ The Okta PAM Terraform Provider is used to interact with Okta's Advanced Server
 - Adds support for configuring Active Directory connections for supporting AD Joined user authentication.
 - Data sources are now available for AD connections, Gateway Setup and Server Enrollment tokens and Projects/Project Groups.
 
-Additionally, this provider adds support for specific Beta features. Users should not use resources marked as 'Beta' in production environments and before confirming that the Beta feature has been enabled for your ASA team.
+Additionally, this provider adds support for specific Beta features. Users should not use resources marked as 'Beta' in production environments and before confirming that the Beta feature has been enabled for your OPA/ASA  team.
```
[the remark this answers is not in the document]
Comment (annejuan-okta) [reply docs/index.md:20]: Team or org?

File: templates/index.md.tmpl [hunk 2]
```diff
@@ -15,14 +17,14 @@ The Okta PAM Terraform Provider is used to interact with Okta's Advanced Server
...
+Additionally, this provider adds support for specific Beta features. Users should not use resources marked as 'Beta' in production environments and before confirming that the Beta feature has been enabled for your OPA/ASA  team.
 
 ## Authentication
-The Okta PAM Providers requires a Service User account that is granted the 'Admin' role be created. This Service User account will be used by the Provider to authenticate to ASA.
+The Okta PAM Providers requires a Service User account that is granted the 'Admin' role be created. This Service User account will be used by the Provider to authenticate to OPA/ASA.
 
 
-1) Follow the [guide](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/service-users.htm) to create a Service User account with Admin permissions
-- Use your ASA Team name (same team that the service account in step 1 resides in) as the value for the 'oktapam_team' value.
+1) Follow the [guide](https://help.okta.com/en-us/content/topics/privileged-access/pam-service-users.htm) to create a Service User account with Admin permissions
```
Comment (annejuan-okta) [inline templates/index.md.tmpl:26]:
```suggestion
1) Follow the [guide](https://help.okta.com/en-us/content/topics/privileged-access/pam-service-users.htm) to create a service user account with admin permissions.
```

File: templates/index.md.tmpl [hunk 3]
```diff
@@ -15,14 +17,14 @@ The Okta PAM Terraform Provider is used to interact with Okta's Advanced Server
...
+- Use your OPA/ASA Team name (same team that the service account in step 1 resides in) as the value for the 'oktapam_team' value.
 - Use the Service Account ID for the 'oktapam_key' value
```
Comment (annejuan-okta) [inline templates/index.md.tmpl:28]:
```suggestion
- Use the service account ID for the 'oktapam_key' value.
```

File: templates/index.md.tmpl [hunk 4]
```diff
@@ -15,14 +17,14 @@ The Okta PAM Terraform Provider is used to interact with Okta's Advanced Server
...
 - Use the Service Account key for the 'oktapam_secret' value.
```
Comment (annejuan-okta) [inline templates/index.md.tmpl:29]:
```suggestion
- Use the service account key for the 'oktapam_secret' value.
```

File: docs/index.md [hunk 5]
```diff
@@ -15,14 +17,14 @@ The Okta PAM Terraform Provider is used to interact with Okta's Advanced Server
...
-The Okta PAM Providers requires a Service User account that is granted the 'Admin' role be created. This Service User account will be used by the Provider to authenticate to ASA.
+The Okta PAM Providers requires a Service User account that is granted the 'Admin' role be created. This Service User account will be used by the Provider to authenticate to OPA/ASA.
 
 
-1) Follow the [guide](https://help.okta.com/asa/en-us/Content/Topics/Adv_Server_Access/docs/service-users.htm) to create a Service User account with Admin permissions
-- Use your ASA Team name (same team that the service account in step 1 resides in) as the value for the 'oktapam_team' value.
+1) Follow the [guide](https://help.okta.com/en-us/content/topics/privileged-access/pam-service-users.htm) to create a Service User account with Admin permissions
+- Use your OPA/ASA Team name (same team that the service account in step 1 resides in) as the value for the 'oktapam_team' value.
 - Use the Service Account ID for the 'oktapam_key' value
 - Use the Service Account key for the 'oktapam_secret' value.
```
Comment (annejuan-okta) [inline docs/index.md:29]:
```suggestion
- Use the service account key for the 'oktapam_secret' value.
```

File: docs/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SajanAlexander-okta) [reply docs/index.md:20]: team

[the remark this answers is not in the document]
Comment (SajanAlexander-okta) [reply docs/index.md:20]: will commit the suggestion, but then change it to team

**Pull request merged**